core/src/main/java/org/apache/accumulo/core/client/impl/Namespace.java

```diff
@@ -35,6 +37,7 @@
    * Uses an internal cache and private constructor for storing a WeakReference of every Namespace.ID. Therefore, a Namespace.ID can't be instantiated outside
    * this class and is accessed by calling Namespace.ID.{@link #of(String)}.
    */
+  @XmlJavaTypeAdapter(JaxbAbstractIdSerializer.class)
```

Comment:
Too bad `@XmlJavaTypeAdapter` is not marked with `@Inherited`. Then, you could just annotate `AbstractId` instead of both `Table.ID` and `Namespace.ID`. It may be possible to register the serializer directly, using a method call in the monitor code instead of annotations. This would also save from adding the compile-time dependency to the core module. But, this is almost certainly not worth it.